lib/components/user/places/place-editor.tsx

```diff
@@ -251,6 +252,7 @@ class PlaceEditor extends Component<Props> {
 
             <PlaceLocationField
               className="form-control"
+              geocoderResultsOrder={geocoderResultsOrder}
```

Comment:
`geocoderResultsOrder` is already included in `geocoderConfig` which is already passed in! Can't we just pull it from there instead of adding a new prop?

Reply:
Good catch! Updated in [0045ca5](https://github.com/opentripplanner/otp-react-redux/pull/1385/commits/0045ca5870c7f895a65e7d26f5cc5676522f511e)